failure looks real:
```
ERROR: test_force_consumption (corehq.apps.commtrack.tests.test_xml.CommTrackOTATest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 232, in inner
    return helper(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 205, in __call__
    call_with_settings(fn_with_pre_and_post, run_config.settings, args, kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 220, in call_with_settings
    fn(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 200, in fn_with_pre_and_post
    run_config.pre_run(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/form_processor/tests/utils.py", line 127, in <lambda>
    pre_run=lambda *args, **kwargs: args[0].setUp(),
  File "/mnt/commcare-hq-ro/corehq/apps/commtrack/tests/test_xml.py", line 50, in setUp
    super(CommTrackOTATest, self).setUp()
  File "/mnt/commcare-hq-ro/corehq/apps/commtrack/tests/util.py", line 173, in setUp
    self.users = [bootstrap_user(self, **user_def) for user_def in self.user_definitions]
  File "/mnt/commcare-hq-ro/corehq/apps/commtrack/tests/util.py", line 84, in bootstrap_user
    user.set_location(setup.loc)
  File "/mnt/commcare-hq-ro/corehq/apps/users/models.py", line 1659, in set_location
    self.create_location_delegates([location])
  File "/mnt/commcare-hq-ro/corehq/apps/users/models.py", line 1800, in create_location_delegates
    self.clear_location_delegates()
  File "/mnt/commcare-hq-ro/corehq/apps/users/models.py", line 1781, in clear_location_delegates
    mapping = self.get_location_map_case()
  File "/mnt/commcare-hq-ro/corehq/apps/users/models.py", line 1831, in get_location_map_case
    return CaseAccessors(self.domain).get_case(location_map_case_id(self))
  File "/mnt/commcare-hq-ro/corehq/apps/commtrack/util.py", line 224, in location_map_case_id
    return uuid.uuid5(const.MOBILE_WORKER_UUID_NS, user.user_id)
  File "/usr/local/lib/python2.7/uuid.py", line 589, in uuid5
    hash = sha1(namespace.bytes + name).digest()
UnicodeDecodeError: 'ascii' codec can't decode byte 0xa3 in position 1: ordinal not in range(128)
```

--- a/corehq/apps/commtrack/util.py
+++ b/corehq/apps/commtrack/util.py
@@ -221,7 +221,10 @@ def submit_mapping_case_block(user, index):
 
 def location_map_case_id(user):
     if should_use_sql_backend(user.domain):
-        return uuid.uuid5(const.MOBILE_WORKER_UUID_NS, user.user_id)
+        user_id = user.user_id
+        if isinstance(user_id, unicode):
+            user_id = user_id.encode('utf8')
+        return uuid.uuid5(const.MOBILE_WORKER_UUID_NS, user_id)
     return 'user-owner-mapping-' + user.user_id
 
 

--- a/corehq/apps/commtrack/util.py
+++ b/corehq/apps/commtrack/util.py
@@ -224,7 +224,7 @@ def location_map_case_id(user):
         user_id = user.user_id
         if isinstance(user_id, unicode):
             user_id = user_id.encode('utf8')
-        return uuid.uuid5(const.MOBILE_WORKER_UUID_NS, user_id)
+        return uuid.uuid5(const.MOBILE_WORKER_UUID_NS, user_id).hex
     return 'user-owner-mapping-' + user.user_id
 
 

